src/com/palmergames/bukkit/towny/TownyMessaging.java

```diff
@@ -108,22 +98,30 @@ public static void sendMsg(String msg) {
 	 * @param msg the message being sent
 	 */
 	public static void sendMsg(Object sender, String msg) {
-		for (String line : ChatTools.color(TownySettings.getLangString("default_towny_prefix") + Colors.Green + msg)) {
-			if (sender instanceof Player) {
-				((Player) sender).sendMessage(line);
-			} else if (sender instanceof CommandSender) {
-				((CommandSender) sender).sendMessage(Colors.strip(line));
-			} else if (sender instanceof Resident) {
-				Player p = TownyAPI.getInstance().getPlayer((Resident) sender);
-				if (p == null) {
-					return;
-				}
-				p.sendMessage(Colors.strip(line));
-			}
+		// I don't fully understand the reason for the Resident check but I don't want to break something so...
+		if (sender instanceof Resident) {
+			Player p = TownyAPI.getInstance().getPlayer((Resident) sender);
+			if (p == null)
+				return;

```

Comment:
This if-statement call is highly unlikely (but still valid). However, I wouldn't return on null (bad, in this instance, for control flow and you would skip the dev message) and would instead use a simple `!= null` check